clicache/integration-test/DefaultCacheableN.cs

```diff
@@ -156,29 +132,13 @@ public object CObject
 
     #region IDataSerializable Members
 
-    public Int32 ClassId
+    public String Type
```

Comment:
Now that the C++ doesn’t lookup this type anymore can’t we just used GetType on Object?

Reply:
Indeed we can. Good catch.

Done.